README.md

```diff
@@ -93,4 +93,25 @@ To clean a `slm` package, use `slm clean`. This removes any fetched dependencies
 ## Working directly with `stanza`
 
 You can also call `stanza build` or `stanza repl` directly once you have
-resolved dependencies once using `slm build` or `slm repl`.
\ No newline at end of file
+resolved dependencies once using `slm build` or `slm repl`.
+
+# Building a Conan2 Package
+
+To build the Conan2 package for binary distribution of `slm`:
+
+1.  Build the `slm` utility as described above.
+1.  Setup the Python Environment
+    1.  Use python 3.11 or higher
+    2.  `python3 -m venv venv`
+    3.  `source venv/bin/activate`
+    4.  `pip install -r requirements.txt`
+2.  Run the conan build:
+    1.  `conan create . -s os="Macos" -s arch="x86_64"`
```

Comment:
Running from scratch, I get this conan error at this point.
```
$ conan create . -s os=Linux -s arch=x86_64
ERROR: The default build profile '/home/jwatson/git/slm/.conan2/profiles/default' doesn't exist.
You need to create a default profile (type 'conan profile detect' command)
or specify your own profile with '--profile:build=<myprofile>'
```
Maybe step 2.1 should be:
```
1. set up your conan environment if you haven't already
export CONAN_HOME=$PWD/.conan2
conan profile detect
```

Reply:
Great - I'll update with that!